helix-view/src/gutter.rs

```diff
@@ -13,7 +13,8 @@ fn count_digits(n: usize) -> usize {
     std::iter::successors(Some(n), |&n| (n >= 10).then_some(n / 10)).count()
 }
 
-pub type GutterFn<'doc> = Box<dyn FnMut(usize, bool, bool, &mut String) -> Option<Style> + 'doc>;
+pub type GutterFn<'doc> =
+    Box<dyn FnMut(Option<usize>, bool, bool, &mut String) -> Option<Style> + 'doc>;
```

Comment:
I actually don't think we want ti change the parameters here. It seems like the `Nine` case is ignored everywhere in that case it would be better yo just not render the gutter at all